File: Data Structures & Algorithms/remove-element/submission-5.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int k = 0;
        int i = 0;
        while (i < nums.length){
           int temp = nums[i];
           nums[i] = nums[k];
           nums[k] = temp;
           if(nums[k] != val){
            k++;
           }
           i++;
        }
        return k;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-5.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        compliment_index = {}
        for i, num in enumerate(nums):
            if num in compliment_index:
                return [compliment_index[num], i]
            compliment = target - num
            compliment_index[compliment] = i
            
        return []

        